src/managers/session/authenticated.ts

```diff
@@ -254,4 +269,53 @@ export const isIdentityConsistencyChecked = async (req: Request) => {
     "official_contact_email",
     "bypassed",
   ].includes(link?.verification_type ?? "");
+
+  return match({
+    isOrganizationExecutive: link.is_executive,
+    askForExecutiveCertification,
+    hasValidVerificationType,
+  })
+    .with(
+      {
+        askForExecutiveCertification: true,
+        isOrganizationExecutive: true,
+      },
+      { hasValidVerificationType: true },
+      () => true,
+    )
+    .otherwise(() => false);
 };
+
+export async function getCurrentAcr(req: Request) {
+  const { certificationDirigeantRequested } = req.session;
+  const isConsistencyChecked = await isIdentityConsistencyChecked(req);
```

Comment:
todo: as seen together, getCurrentAcr needs to replace isIdentityConsistencyChecked and contain all the logic. See https://github.com/numerique-gouv/proconnect-identite/pull/1134